sys-cluster/resource-agents/resource-agents-4.0.1.ebuild

```diff
@@ -1,59 +0,0 @@
-# Copyright 1999-2021 Gentoo Authors
```

Comment:
Please don't rename the patch unless it's changing.

Reply:
Oh, it was for convenience